server/src/db/migrations/20220203144345_createReviews.cjs

```diff
@@ -0,0 +1,30 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable('reviews', (table) => {
```

Comment:
Does a user have to be signed in to create a review? Or can visitors of the application write a review?